Tiltfile

```
@@ -327,6 +327,7 @@ def deploy_observability():
     k8s_resource(workload = "promtail", extra_pod_selectors = [{"app": "promtail"}], labels = ["observability"])
     k8s_resource(workload = "loki", extra_pod_selectors = [{"app": "loki"}], labels = ["observability"])
     k8s_resource(workload = "grafana", port_forwards = "3000", extra_pod_selectors = [{"app": "grafana"}], labels = ["observability"])
+    k8s_resource(workload = "prometheus-server", port_forwards = "9090", extra_pod_selectors = [{"app": "prometheus"}], labels = ["observability"])
```

can this be simply prometheus?
```suggestion
    k8s_resource(workload = "prometheus", port_forwards = "9090", extra_pod_selectors = [{"app": "prometheus"}], labels = ["observability"])
```

---

I think this has to match the name of the Deployment in the helm chart. Independent of that we can call the identifier for the tilt-settings "prometheus" (after the rebase)

---

@sbueringer is right, the `workload` argument must match the deployment in the helm chart, and that always includes a suffix.

I was able to change the displayed name in the tilt ui to prometheus by using the `new_name = "prometheus"` argument to this method.